Description: Install files in the right location


--- dedisp-ajameson-0+git20170217.orig/Makefile.inc
+++ dedisp-ajameson-0+git20170217/Makefile.inc
@@ -1,13 +1,13 @@
 # The definitions in this file should be changed to match your system
 
-CUDA_PATH ?= /usr/local/cuda
-INSTALL_DIR = $(HOME)/linux_64
+CUDA_PATH ?= /usr/
+INSTALL_DIR = $(DESTDIR)/usr
 
 CUDA_DIR   ?= $(CUDA_PATH)
 THRUST_DIR ?= $(CUDA_DIR)/include
 
 # LIB_ARCH   = lib
-LIB_ARCH   = lib64
+LIB_ARCH   = lib
 # GPU_ARCH   = sm_13
 # GPU_ARCH   = sm_20
 GPU_ARCH   = sm_30
